Frontend/src/packages/editor/components/buttons/EditorSelectFont.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useSlate } from "slate-react";
+import { Editor } from "slate";
+
+const selectFont = () => {
+    const editor = useSlate();
+
+    return (
+        <select name="fonts" id="fontDropdown" onChange={(event) => {
+            Editor.addMark(editor, "textSize", event.currentTarget.value)
+        }}>
```

Comment:
add typescript Types for event, it should be `React.ChangeEvent`, should also add null checkes to reduce runtime errors

```
const val = event.currentTarget?.value;
if(!val) return;
```